### DISCUSSION TITLE
Setup on IIS

### DISCUSSION THREAD

**Discussion by brown-matt-att-net** [gogs/gogs#7589] [Q&A]
I installed the application on my Windows machine and would like to have it accessible via IIS as an application. I tried to setup a reverse proxy to the localhost:3000 but I have been unsuccessful. Is it possible to use the url rewrite rules to have it work off of /git as I have other applications running on the same website?

**Comment by Bikunov-Alexey** [comment]
yes.
gogs support rewersproxy iis.
